sections/introduction.tex

```diff
@@ -2,18 +2,18 @@ \chapter{Introduction} % (fold)
 \label{cha:introduction}
 Ever since the dawn of mathematics, mathematicians has strived for faster computations while maintaining correctness.
 In our time computers have become the main target for doing extensive computations, and their CPUs are what the majority of programming languages apply to.
```

Comment:
Jeg tænker det er fint med their, fordi der før kommaet tales om computeren :) Men man kan vel også bare skrive the.